aws/load_balancer/waf.tf

```diff
@@ -208,6 +208,43 @@ resource "aws_wafv2_web_acl" "forms_acl" {
     (var.billing_tag_key) = var.billing_tag_value
     Terraform             = true
   }
+
+  rule {
+    name     = "BlockInvalidURLPath"
+    priority = 7
+
+    action {
+      block {}
+    }
+
+
+    statement {
+      not_statement {
+        statement {
+          regex_pattern_set_reference_statement {
```

Comment:
Noticed that we're using a regex_pattern_set but only comparing against a single regex string.  I'm wondering if less AWS WAF Web ACL capacity units (WCU) would be used if we change to single regex match rule statement instead?